src/shared/components/mutationTable/column/FunctionalImpactColumnFormatter.tsx

```diff
@@ -66,7 +66,7 @@ class FunctionalImpactColumnTooltip extends React.Component<
                     <thead>
                         <tr>
                             <th>Legend</th>
-                            {SHOW_MUTATION_ASSESSOR && (
+                            {shouldShowMutationAssessor() && (
```

Comment:
I wouldn't call this function multiple times. How about defining a constant before the return statement and using it everywhere else?

```JavaScript
    const showMutationAssessor = shouldShowMutationAssessor();
```
```suggestion
                            {showMutationAssessor && (
```